app/utils/configureStore.js

```diff
@@ -50,26 +51,43 @@ const trackerMiddleware = createTracker({
   }
 });
 
-Raven.config(config.ravenDSN, {
-  release: config.release,
-  environment: config.environment
-}).install();
+const ravenMiddlewareOptions = {
+  stateTransformer: state => {
+    try {
+      const token = jwtDecode(state.auth.token);
+      return {
+        ...state,
+        auth: {
+          ...state.auth,
+          token
+        }
+      };
+    } catch (e) {
+      return state;
+    }
+  },
+  getUserContext: state => omit(selectCurrentUser(state), 'icalToken')
+};
 
 const loggerMiddleware = createLogger({
   level: 'info',
   collapsed: true
 });
 
-const messageMiddleware = createMessageMiddleware(message =>
-  addToast({ message })
-);
+export default function configureStore(
+  initialState: State | {||},
+  Raven: any
```

Comment:
https://github.com/flowtype/flow-typed/blob/master/definitions/npm/raven-js_v3.17.x/flow_v0.38.x-/raven-js_v3.17.x.js

Reply:
Yeah, tried to use that one.. But it has terrible typeings, together with function signatures that returns `this`. It is therefore only compatible with "itself", and cannot be used as an interface for a generic raven client (one for ssr/node and one for client).